src/configurations/sources/facebook_lead_ads_native/db-config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "facebook_lead_ads_native",
+  "category": "webhook",
+  "displayName": "Facebook Lead Ads Native",
+  "config": {
+    "auth": {
+      "type": "OAuth"
+    },
+    "supportedAccountDefinitions": {
+      "rudderAccountId": ["SOURCE_FACEBOOK_LEAD_ADS_NATIVE_OAUTH"]
+    }
+  },
+  "options": {
```

Comment:
I believe we should also use a feature flag to control the rollout to the customers.
CC @krishna2020